### PR TITLE
Improved naming of discovered findings filter and add same for mitigated

### PR DESCRIPTION
**Pull request by quirinziessler**
This PR updates the naming of filters related to #10401 to be a bit more precise.
Also it adds the same filter options for the "mitigated" field.

### CODE REVIEW

**Comment by manuel-sommer** [comment]
Hi @mtesauro, could you take a look here? It would be nice if we can merge this for the next release :-)

File: dojo/filters.py [hunk 1]
```diff
@@ -1663,6 +1667,8 @@ def set_date_fields(self, *args: list, **kwargs: dict):
         self.form.fields["on"].widget = date_input_widget
         self.form.fields["before"].widget = date_input_widget
         self.form.fields["after"].widget = date_input_widget
+        date_mitigated_input_widget = forms.DateInput(attrs={"class": "datepicker", "placeholder": "YYYY-MM-DD"}, format="%Y-%m-%d")
+        self.form.fields["on"].widget = date_mitigated_input_widget
```
Comment (Maffooch) [inline dojo/filters.py:1671]:
```suggestion
        self.form.fields["mitigated_on"].widget = date_input_widget
        self.form.fields["mitigated_before"].widget = date_input_widget
        self.form.fields["mitigated_after"].widget = date_input_widget        
```
The `date_input_widget` could be reused from above. No need to redefine the same thing twice

**Review by Maffooch — CHANGES_REQUESTED** [review]
@quirinziessler thank you for your patience here! I left a few comments for moving forward and getting these changes merged 😄

**Comment by quirinziessler** [comment]
@Maffooch added your suggestions to the MR. Sorry for the late reaction. Should be good to go from now.

File: dojo/filters.py [hunk 2]
```diff
@@ -1543,18 +1546,21 @@ def filter(self, qs, value):
 
 class FindingFilterHelper(FilterSet):
     title = CharFilter(lookup_expr="icontains")
-    date = DateRangeFilter(field_name="date", label="Date Discovered")
-    on = DateFilter(field_name="date", lookup_expr="exact", label="On")
-    before = DateFilter(field_name="date", lookup_expr="lt", label="Before")
-    after = DateFilter(field_name="date", lookup_expr="gt", label="After")
+    date = DateFromToRangeFilter(field_name="date", label="Date Discovered")
```
Comment (Maffooch) [inline dojo/filters.py:1549]:
```suggestion
    date = DateRangeFilter(field_name="date", label="Date Discovered")
```
This change was reverted back to `DateRangeFilter` in the #10760 to accommodate date changes that are relative (i.e. X days ago) to today

**Review by Maffooch — APPROVED** [review]

**Review by cneill — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Review by blakeaowens — APPROVED** [review]

**Pull request merged**